### PR TITLE
Added example for rp2040

### PR DESCRIPTION
**Pull request by Rocketct**

### CODE REVIEW

**Review by per1234 — CHANGES_REQUESTED** [review]
Since it was more convenient in this particular case than a review suggestion, I submitted a PR to your fork with the necessary CI workflow adjustments:
https://github.com/Rocketct/Arduino_ScienceJournal/pull/1
If that PR is merged, it will be added in to this PR.

File: examples/RP2040ConnectFirmware/RP2040ConnectFirmware.ino [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+  }
+
+  PDM.onReceive(onPDMdata);
+  if (!PDM.begin(1, 16000)) {
+    printSerialMsg("Failed to start PDM!");
+    blinkLoop();
+  }
+
+  if (!BLE.begin()) {
+    printSerialMsg("Failed to initialized BLE!");
```
Comment (per1234) [inline examples/RP2040ConnectFirmware/RP2040ConnectFirmware.ino:103]:
```suggestion
    printSerialMsg("Failed to initialize BLE!");
```

File: examples/RP2040ConnectFirmware/RP2040ConnectFirmware.ino [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  while (!Serial);
+  Serial.println("Started");
+  #endif
+
+  delay(2000);
+
+  pinMode(RESISTANCE_PIN, INPUT); // Used for reading resistance
+
+  if (!IMU.begin()) {
+    printSerialMsg("Failed to initialized IMU!");
```
Comment (per1234) [inline examples/RP2040ConnectFirmware/RP2040ConnectFirmware.ino:92]:
```suggestion
    printSerialMsg("Failed to initialize IMU!");
```

**Review by per1234 — CHANGES_REQUESTED** [review]

**Review by per1234 — APPROVED** [review]
I don't have enough knowledge about this project to be able to effectively review the pull request as a whole, but as far as the CI update and grammar nitpickery goes I'm satisfied.

**Comment by Rocketct** [comment]
> I don't have enough knowledge about this project to be able to effectively review the pull request as a whole, but as far as the CI update and grammar nitpickery goes I'm satisfied.

yes @per1234  but this is enough :smile:  is exactly what we need for this PR, this example was added just as a proof of concept that shows how to use the SJ with RP2040  :+1:

**Pull request merged**